GUI/MainWindow.xaml.cs

```diff
@@ -1409,5 +1412,18 @@ private void AddContaminantXML_Click(object sender, RoutedEventArgs e)
             }
             dataGridProteinDatabases.Items.Refresh();
         }
+
+        // handle window closing
+        private void MainWindow_Closing(object sender, CancelEventArgs e)
+        {
+            if (!GuiGlobalParams.DisableCloseWindow)
+            {
+                e.Cancel = true;
+                var exit = MessageBox.Show("Are you sure?", "Exit MetaMorpheus", MessageBoxButton.YesNo, MessageBoxImage.Question);
+
+                if (exit == MessageBoxResult.Yes)
+                    e.Cancel = false;
```

Comment:
put brackets around ifs, elses, etc, even if it's only one line. just a code style thing